Trace/Metric/Resource.swift

```diff
@@ -38,7 +38,6 @@ struct Resource: Codable {
     let platform: String = "iOS"
```

Comment:
CodingKeys country key should be removed right?

Reply:
Thank you. Updated.